src/puppetlabs/puppetdb/http/aggregate_event_counts.clj

```diff
@@ -6,14 +6,15 @@
             [puppetlabs.puppetdb.query-eng :refer [produce-streaming-body]]
             [puppetlabs.puppetdb.middleware :refer [verify-accepts-json validate-query-params]]
             [clojure.tools.logging :as log]
-            [net.cgrand.moustache :refer [app]]))
+            [net.cgrand.moustache :refer [app]]
+            [puppetlabs.puppetdb.config :as conf]))
 
 (defn routes
-  [version]
+  [version config]
   (app
     [""]
     {:get (fn [{:keys [params globals]}]
-            (when (= "puppetdb" (:product-name globals))
+            (when (conf/foss? config)
               (log/warn "The aggregate-event-counts endpoint is experimental"
                         " and may be altered or removed in the future."))
```

Comment:
existing issue, but we're logging this line twice: here and on line 32 below. Also, I think log/warn is handling the spacing for you. I see this rendered as:

```
The aggregate-event-counts endpoint is experimental  and may be altered or removed in the future.
```

and the same on the event-counts meesage